README.md

```diff
@@ -416,8 +416,12 @@ Special values:
 
 * `:required => true` Will display a red '*' to show it's required
 * `:scope => :the_scope` Will scope parameters in the hash, scoping can be nested. See example
+* `:method => :method_name` Will use specified method as a parameter value
 
-The value of scoped parameters can be set with both scoped (`let(:order_item_item_id)`) and unscoped (`let(:item_id)`) methods. It always searches for the scoped method first and falls back to the unscoped method.
+Retrieving of parameter value goes through several steps:
+1. if `method` option is defined and test case responds to this method then this method is used;
+2. if test case responds to scoped method then this method is used;
+3. overwise unscoped method is used.
```

Comment:
Your example is working as intended. If instead the `example` is an `example_request` then we hit your failure, which does exist. Status is a weird method to override, and we've always had an issue with it.

The problem line seems like it would be [this one](https://github.com/zipmark/rspec_api_documentation/pull/312/files#diff-fdcc870847d867c80b7b6564711aecb2R49). I will look at a PR if you want to fix it, otherwise I don't see why you would change the method and not define it.